lib/Linting.js

```diff
@@ -300,14 +300,21 @@ Linting.prototype._createElementIssues = function(elementId, elementIssues) {
     return;
   }
 
+  // Also attach element to subprocess plane
+  var plane = this._elementRegistry.get(elementId + '_plane');
+  if (plane) {
+    this._createElementIssues(plane.id, elementIssues);
+  }
+
   var menuPosition;
   var position;
 
-  if (element === this._canvas.getRootElement()) {
+  var isRoot = !element.parent;
+  if (isRoot) {
     menuPosition = 'bottom-right';
 
     position = {
-      top: 20,
+      top: 50,

```

Comment:
Is this to account for a potential breadcrumb?

![image](https://user-images.githubusercontent.com/58601/155289133-0bdf6ea5-ccf1-4281-b6c7-1538b7f9a378.png)

What about making it adaptable (i.e. react to weather the [`.bjs-breadcrumbs-shown`] is there).

Reply:
Good point, I'll add that! The original PR was created before we had proper css classes for that and I did not think about it when rebasing it

Reply:
Reacting to the breadcrumb css felt a bit clunky when all overlays are positioned absolutely. I added the cases:
  - root is process
  - root is SubProcess

as separate options with 81b140bd217d77ddf8331c7ec6f197df9cf2fa55